cclient/serialization/json/json_serializer.c

```diff
@@ -70,6 +70,22 @@ retcode_t json_array_to_int_array_array(cJSON* obj, const char* obj_name,
   return RC_OK;
 }
 
+retcode_t json_boolean_array_to_utarray(cJSON* obj, const char* obj_name,
+                                        UT_array* ut) {
+  cJSON* json_item = cJSON_GetObjectItemCaseSensitive(obj, obj_name);
+  if (cJSON_IsArray(json_item)) {
+    cJSON* current_obj = NULL;
+    /* The type of json object is expected to be boolean type */
+    cJSON_ArrayForEach(current_obj, json_item) {
+      utarray_push_back(ut, &current_obj->type);
```

Comment:
1. `type` is a bit-flag. 
2. please check object type, if not boolean return `RC_CCLIENT_JSON_PARSE` in advance.